
This patch fixes arches like PowerPC which do default to unsigned char.

  - Rene Rebe <rene@rocklinux.org>

--- netcat-0.6.1/src/flagset.c.orig	2002-10-03 12:25:16.000000000 +0200
+++ netcat-0.6.1/src/flagset.c	2004-06-28 23:00:40.529869256 +0200
@@ -134,7 +134,7 @@
 
 int netcat_flag_count(void)
 {
-  register char c;
+  register signed char c;
   register int i;
   int ret = 0;
 
